.github/workflows/packages-contracts.yml

```
@@ -36,3 +41,61 @@ jobs:
+        uses: actions/setup-dotnet@v5
+        with:
+          dotnet-version: '9.0.x'
+          source-url: https://nuget.pkg.github.com/phoenixvc/index.json
+        env:
+          NUGET_AUTH_TOKEN: ${{ secrets.GH_PACKAGES_TOKEN }}
+
+      - name: Restore packages
+        run: dotnet restore packages/contracts/dotnet/Mystira.Contracts
+
+      - name: Check for vulnerable packages
+        run: |
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2086:info:10:17: Double quote to prevent globbing and word splitting

(shellcheck)